components/formats-gpl/src/loci/formats/in/LiFlimReader.java

```diff
@@ -231,7 +231,7 @@ else if (TypeUINT12) {
       int thisSeries = getSeries();
       for (int i=0; i<thisSeries; i++) {
         setSeries(i);
-        in.skipBytes(getImageCount() * FormatTools.getPlaneSize(this));
+        in.skipBytes((long) getImageCount() * FormatTools.getPlaneSize(this));
```

Comment:
This seems to have impacted the non-regression tests at https://merge-ci.openmicroscopy.org/jenkins/job/BIOFORMATS-test-folder/52532/console. We might need to look at the data with or without this PR to establish whether the configuration needs to be regenerated or there is an issue with this change.

Reply:
Reasonably certain that the configuration needs to be updated, as proposed here: https://github.com/ome/bioformats/pull/3529#pullrequestreview-375846632

Note that without that change, the MD5s for series 0 and series 1 were the same, which points to this PR fixing a problem.